demos/speech_recognition_wav2vec_demo/python/speech_recognition_wav2vec_demo.py

```diff
@@ -57,17 +57,16 @@ def __init__(self, core, model_path, input_shape, device, vocab_file):
         if len(model.inputs) != 1:
             raise RuntimeError('Wav2Vec must have one input')
         self.input_tensor_name = model.inputs[0].get_any_name()
-        model_input_shape = model.inputs[0].shape
+        model_input_shape = model.inputs[0].partial_shape
```

Comment:
as demo can work with custom models , I think we should also check that partial_shape is dynamic. In future model can be supported with different devices where dynamic shapes are not supported. Also for some scenarios it can be useful to compare demo results with dynamic and static shape, so possibly it can be useful if you allow switching between dynamic and static model using comand line parameter